R/RerF.R

```diff
@@ -93,6 +96,35 @@ RerF <-
 		# na.action = function (...) { Y <<- Y[rowSums(is.na(X)) == 0];  X <<- X[rowSums(is.na(X)) == 0, ] },
 		# @param na.action action to take if NA values are found. By default it will omit rows with NA values. NOTE: na.action is performed in-place. See default function.
 
+    defaults <- function(paramList){
```

Comment:
move to randmat file so it can be documented there

then it can be called with ?rerf:::defaults to see the defaults

Reply:
I found out that it can be called simply with `?defaults` after `rerf` is loaded.